modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,162 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` to allow the cluster to provision the host. 
+
+Existing control plane BareMetalHosts may have the externallyProvisioned flag set to true if they were provisioned by {product-title}.
+====
+
+.Procedure
+
+. Create the `baremetalhost` and `ipmi` secret:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+----
++
+.Example input
+[source,terminal]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: master-3-bmc-secret
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: _<username>_
+  password: _<password>_
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: master-3
+  namespace: openshift-machine-api
+spec:
+  externallyProvisioned: false
+  online: true
+  bootMACAddress: aa:aa:aa:aa:aa:03
+  bmc:
+	address: ipmi://172.22.0.1:6233
+	credentialsName: master-3-bmc-secret
+EOF
```

Comment:
This makes sense to me, but I'll confirm with QE.

Reply:
I don't have a strong opinion either way. To me, either option conveys the method clearly enough.
However, I am wondering if we should find a way to say that the input is an example.

Reply:
To me, they both sound clear. So I think it's just a matter of convention.
However, maybe if we use the concatenated option, we should find a way to say that this is an example?

Reply:
@awolffredhat combining the command and the concatenated file into a single step removes the "example input" heading. I generally prefer to call out examples, but I'm going to update this as Bob suggests, since that's in line with our other documentation.

Reply:
Updated as suggested.